.github/workflows/ci.yml

```diff
@@ -0,0 +1,152 @@
+name: CI
+
+
+on:
```

Comment:
we should also add (or similar daily runs). unfortunately they don't support @daily, see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule

```yaml
on:
  schedule:
    - cron: '0 0 * * *' 
```

Reply:
I can't find anything in the docs but can we enable notifications for these kind of runs?

Reply:
I think if you watch the repo, you'll get notified? (ref https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications#github-actions-notification-options )